approvals/src/main/java/no/sikt/nva/approvals/domain/Approval.java

```diff
@@ -5,9 +5,15 @@
 import java.util.Objects;
 import java.util.UUID;
 
-public record Approval(UUID identifier, Collection<Identifier> identifiers, URI source) {
+public record Approval(UUID identifier, Collection<Identifier> identifiers, URI source, Handle handle) {
```

Comment:
suggersion: to distinguish our and other identifiers, maybe name our one approvalId instead of just identifier?